src/main/java/org/zalando/nakadi/service/EventTypeService.java

```diff
@@ -101,6 +102,7 @@
     private final String etLogEventType;
     private final NakadiAuditLogPublisher nakadiAuditLogPublisher;
     private final EventTypeOptionsValidator eventTypeOptionsValidator;
+    private final JsonSchemaEnrichment jsonSchemaEnrichment = new JsonSchemaEnrichment();
```

Comment:
Isn't it better to add the initialization to the constructor?

Reply:
Done